plugins/magicline/plugin.js

```diff
@@ -570,7 +570,7 @@
 	function initLine( that ) {
 		var doc = that.doc,
 			// This the main box element that holds triangles and the insertion button
-			line = newElementFromHtml( '<span contenteditable="false" data-cke-magic-line="1" style="' + CSS_COMMON + 'position:absolute;border-top:1px dashed ' + that.boxColor + '"></span>', doc ),
+			line = newElementFromHtml( CKEDITOR.tools.htmlSafeByReview('<span contenteditable="false" data-cke-magic-line="1" style="' + CSS_COMMON + 'position:absolute;border-top:1px dashed ' + that.boxColor + '"></span>', 'Template created using safe internal values'), doc ),
```

Comment:
Looks good, let's just double check the boxColor

Reply:
Looks like it is either a constant or an editor.config value. https://sourcegraph.com/github.com/ckeditor/ckeditor4/-/blob/plugins/magicline/plugin.js?L31. Added to justification